schemas/components.cue

```diff
@@ -9,13 +9,27 @@ package components
 	previousLatestVersion?: #ContainerImagePrefetchOptimization
 }
 
+#OSSelector: {
+    os:   string
+    arch: string
+}
+
+
 #ContainerImage: {
 	downloadURL: string
 	amd64OnlyVersions:     [...string]
 	multiArchVersionsV2:   [...#VersionV2]
 }
 
+#GPUContainerImage: {
+	downloadURL: string
+	multiArchVersionsV2:   [...#VersionV2]
```

Comment:
Any recommendations for alternate names for `multiArchVersionsV2`? Thought of `Version` but `VersionV2` already exists

Reply:
`gpuImageVersion`? 
I 'd love to have GPU in capital letters but it's not following the naming convention.

Reply:
thanks, switched to `gpuImageVersion`. Yes, caps would be nice but isn't consistent with the rest of the fields.